.github/workflows/e2e-test-no-domain.yaml

```diff
@@ -86,7 +94,9 @@ jobs:
         working-directory: test/
         run: |
           set -o pipefail
-          aws configure set aws_access_key_id "$AWS_ACCESS_KEY_ID" --profile default && aws configure set aws_secret_access_key "$AWS_SECRET_ACCESS_KEY" --profile default && aws configure set region "$AWS_DEFAULT_REGION" --profile default
+          # boto3 ignores AWS creds env vars for some reason
```

Comment:
Since we're using temp credentials, session token needs to be in credentials file.